src/ckb-daemon/led_keyboard.c

```diff
@@ -127,6 +138,12 @@ int updatergb_kb(usbdevice* kb, int force){
         makergb_full(newlight, data_pkt);
         if(!usbsend(kb, data_pkt[0], 12))
             return -1;
+    } else if (IS_K55(kb)) {
```

Comment:
I suggest moving this to the top of the if, so that we don't have to do `&& !IS_K55(kb)` above.